dali/pipeline/util/backend2workspace_map.h

```diff
@@ -62,6 +62,60 @@ class Backend2WorkspaceMap<MixedBackend> {
 template<typename Backend>
 using Workspace = typename Backend2WorkspaceMap<Backend>::Type;
 
+// Actual trait-conventions used, maps as above with exception of CPUBackend -> HostWorkspace
+template <typename Backend>
```

Comment:
I would be good to update `Backend2WorkspaceMap` in the next PR as well.

Reply:
Kind of done.